src/F500/CI/Console/Command/QueueWorkerCommand.php

```diff
@@ -24,44 +26,52 @@ protected function configure()
     {
         $this
             ->setName('queue:worker')
-            ->setDescription('Starts a worker for the job-queue.');
+            ->setDescription('Starts a worker for the job-queue.')
+            ->addOption(
+                'tube',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'The tube to push this job to.',
+                'ci-builds'
+            );
 
         parent::configure();
     }
 
     /**
-     * @param InputInterface  $input
+     * @param InputInterface $input
      * @param OutputInterface $output
      * @return void
      * @throws \RuntimeException
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $app     = $this->getContainer();
+        $app = $this->getContainer();
         $rootDir = $app['root_dir'];
 
         /** @var \F500\CI\Command\Process\ProcessFactory $processFactory */
         $processFactory = $app['f500ci.process_factory'];
 
         while (true) {
+            /* @var Job $job */
             $job = $this->pheanstalk
-                ->watch('ci-builds')
+                ->watch($input->getOption('tube'))
                 ->reserve();
 
             $payload = json_decode($job->getData(), true);
 
             if (!$this->isPayloadValid($payload)) {
-                $output->writeln(sprintf('Buried job <comment>%s</comment> because of an invalid payload.', $job->getId()));
+                $output->writeln(sprintf('Buried job <comment>%s</comment> because of an invalid payload.',
+                    $job->getId()));
                 $this->pheanstalk->bury($job);
 
             } else {
                 $args = array('exec', 'app/console', '--ansi', 'run', $payload['suite']);
 
                 if (!empty($payload['params'])) {
-                    foreach ($payload['params'] as $key => $value) {
-                        $args[] = "{$key}:{$value}";
-                    }
+                    $args[] = "--build_info " . base64_encode(json_encode($payload['params']));
```

Comment:
Can you change this to `build-info`? 😄 (The argument then becomes `--build-info`)